run.go

```diff
@@ -5,14 +5,24 @@ import (
 	"github.com/moby/buildkit/identity"
 )
 
+// RunOptions is a helper wrapper to return a list of
+// RunOptions as a single llb.RunOption
```

Comment:
`s/RunOptions/llb.RunOptions/`

Add a `.` at the end, also for `SetRunOption` below.